Description: Avoid to exec configure twice
Upstream autogen.sh carries a fix configure that can't handle (*)FLAGS
Author: Dererk <dererk@debian.org>
Last-Update: <2010-08-20>

--- libfprint-20100820git.orig/autogen.sh
+++ libfprint-20100820git/autogen.sh
@@ -4,5 +4,3 @@ aclocal || exit 1
 autoheader || exit 1
 autoconf || exit 1
 automake -a -c || exit 1
-./configure --enable-maintainer-mode --enable-examples-build \
-	--enable-x11-examples-build --enable-debug-log $*
